api/src/main/java/io/strimzi/api/kafka/model/bridge/KafkaBridgeSpec.java

```diff
@@ -77,15 +80,28 @@ public void setReplicas(int replicas) {
     }
 
     @JsonInclude(JsonInclude.Include.NON_NULL)
-    @Description("Enable the metrics for the Kafka Bridge. Default is false.")
+    @Description("Enable the metrics for the Kafka Bridge. Default is false. Deprecated, use `spec.metricsConfig`.")
```

Comment:
I guess you should add the `@Deprecated`, `@DeprecatedProperty(description="...", movedToPath="...") and `@PresentInVersions` annotations as well. IIRc that should also automatically generate the docs so you might want to revert the description change.